compat/visual-composer/siteorigin_widget_vc_template.php

```diff
@@ -0,0 +1,5 @@
+<?php
+$widget_settings = $this->get_widget_settings( $atts );
+$instance = $this->update_widget( $widget_settings['widget_class'], $widget_settings['widget_data'] );
```

Comment:
Does this mean we're going to be running the widget through the update function on every page load?